src/pages/Liquidated.tsx

```diff
@@ -29,50 +22,31 @@ export function Liquidated() {
   };
 
   //  Queries for graph
-  const { data: tokenNamesData } = useSWR<AxiosResponse, AxiosError>(LIQUIDATION_GRAPH_TOKENS_QUERY, subQueryFetcher);
-  const tokenNamesResponse: TokenNamesResponse = tokenNamesData?.data.data;
-  const tokenNames = tokenNamesResponse?.vaultManagerMetrics.nodes.map((node) => node.liquidatingCollateralBrand) || [];
-
-  const { data: dailyMetricsData, isLoading: graphDataIsLoading } = useSWR<AxiosResponse, AxiosError>(
-    LIQUIDATION_DAILY_METRICS_QUERY(tokenNames),
-    subQueryFetcher,
+  const { data: vaultStateDailyData, isLoading: graphDataIsLoading } = useSWR<AxiosResponse, AxiosError>(
+    VAULT_STATE_DAILIES_QUERY,
+    (query: string) => subQueryFetcher(query, SUBQUERY_STAGING_URL),
   );
-  const dailyMetricsResponse: LiquidationMetricsDailyResponse = dailyMetricsData?.data.data;
+  const vaultStateDailyResponse: VaultStateDailyResponse = vaultStateDailyData?.data.data;
   const graphDataMap: { [key: number]: GraphData } = {};
-  Object.values(dailyMetricsResponse || {}).forEach((tokenDataList) => {
-    tokenDataList?.nodes.forEach((dailyTokenMetrics) => {
-      graphDataMap[dailyTokenMetrics.dateKey] = {
-        ...graphDataMap[dailyTokenMetrics.dateKey],
-        x: dailyTokenMetrics.blockTimeLast.split('T')[0],
-        key: dailyTokenMetrics.dateKey,
-        active: {
-          ...graphDataMap[dailyTokenMetrics.dateKey]?.active,
-          [dailyTokenMetrics.liquidatingCollateralBrand]: dailyTokenMetrics.numActiveVaultsLast,
-        },
-        liquidated: {
-          ...graphDataMap[dailyTokenMetrics.dateKey]?.liquidated,
-          [dailyTokenMetrics.liquidatingCollateralBrand]: dailyTokenMetrics.numLiquidationsCompletedLast,
-        },
+  vaultStateDailyResponse?.vaultStatesDailies.nodes?.forEach((vaultState) => {
```

Comment:
Add optional chaining operator with `vaultStatesDailies` as well

Reply:
if the `vaultStatesDailies` object exists, the `nodes` object will exist as well